### DISCUSSION TITLE
blog/getting-started

### DISCUSSION THREAD

**Discussion by giscus** [louisescher/spectre#1] [General]
# blog/getting-started

Find out how to configure Spectre.

https://spectre.louisescher.dev/blog/getting-started

**Comment by louisescher** [comment]
This is a comment powered by [giscus](https://giscus.app)! Hope you enjoy this theme :D

**Comment by Zxce3** [comment]
What a nice theme!!!

**Comment by namroj** [comment]
Great theme! Congrats!
